src/components/players/Players/Players.js

```diff
@@ -98,7 +109,92 @@ export default class Players extends Component {
             </table>
           </div>
         </Tab>
+        <Tab label={this.context.t('players.alle')}>
+          <Accordion style={{marginTop: 10}}>
+            <Panel header={this.context.t('players.vttlTeamA')} eventKey="1">
+              {this._getPlayersOfTeam('Vttl','A')}
+            </Panel>
```

Comment:
Dit gaat er vanuit dat er x ploegen zijn...

Om dynamisch te maken zodat het elk jaar juist is:

``` 
this.props.teams.sort((a, b) => (a.competition + a.teamCode).localeCompare(b.competition + b.teamCode))
    .map((team, index) => <Panel ...)
```